vista.html

```diff
@@ -1,317 +0,0 @@
-<!DOCTYPE html>
```

Comment:
could add vista.html to gitignore to avoid this.

Reply:
it already is: https://github.com/carimura/vista/blob/master/.gitignore